### ISSUE TITLE
wrong id in bedrock data ..?

### ISSUE THREAD

**Issue by Zernix2077** [PrismarineJS/minecraft-data#702]
maybe I don’t understand something, but what id are we talking about in bedrock data (https://prismarinejs.github.io/minecraft-data/?v=bedrock_1.19.63)? IDs of blocks and items that minecraft writes to nbt and inventory do not even closely match what is in the documentation, and the commands for issuing these items by ID also give out completely different

Container
![image](https://user-images.githubusercontent.com/110056737/230708723-5799bbb9-fa63-4ab5-89b1-5dd3af6ea4b6.png)

Vanilla NBT with IDs
![image](https://user-images.githubusercontent.com/110056737/230708838-1459166f-9bea-4849-821f-0eed07f34649.png)

MCData IDs
![image](https://user-images.githubusercontent.com/110056737/230708853-907a2aed-f6f8-400a-aa90-903f2df00569.png)
![image](https://user-images.githubusercontent.com/110056737/230708866-6a5093d2-d5d0-4874-8605-08c4f42e90fc.png)
![image](https://user-images.githubusercontent.com/110056737/230708873-97539f2b-70d0-49f5-9d04-c45a0ba16071.png)
![image](https://user-images.githubusercontent.com/110056737/230708881-6ce87065-99b8-46cc-a331-dea654f74e49.png)
![image](https://user-images.githubusercontent.com/110056737/230708885-a4638df8-6497-46e4-b139-033057ce6152.png)
![image](https://user-images.githubusercontent.com/110056737/230708909-cd4b4e6c-2810-41ae-b47a-49ad7c4f5e59.png)
![image](https://user-images.githubusercontent.com/110056737/230708928-8e594ad7-c645-4938-9d0e-028706837d10.png)

**Comment by extremeheat** [comment]
Yes, minecraft-data only have protocol data for versions beyond bedrock 1.19.1. Almost all the other bedrock data is pointing to older data, so they may be out of date. As we update other prismarine- projects to support bedrock, we may add additional data corresponding to newer versions over time.

**Comment by Pupkin1338** [comment]
OK, thank you
I compiled my list of IDs using cores source corresponding to vanilla nbt, checked about 30 random items out of 400 - they all match. I attached it below, maybe it will be useful for you
[bedrockIDs.zip](https://github.com/PrismarineJS/minecraft-data/files/11182835/bedrockIDs.zip)
